sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRelation.scala

```diff
@@ -90,6 +90,19 @@ private[sql] case class JDBCRelation(
 
   override val schema: StructType = JDBCRDD.resolveTable(url, table, properties)
 
+  // Check if JDBCRDD#compileFilter can accept input filters
+  override def unhandledFilters(filters: Array[Filter]): Array[Filter] = {
```

Comment:
This is my personal thought and I do not think we should fix but how about making the structure like `SimpleFilteredScan` in `FilteredScanSuite`? I was working on [those issues](https://issues.apache.org/jira/browse/SPARK-12354) and it would be great if we have the same structure.